### PR TITLE
feat: update tokens for better documentation

### PR DESCRIPTION
**Pull request by alisonailea**
**Related Issue:** #8315

## Summary

This PR updates tokens to include a "calcite-schema" in each token's attributes. This schema fully defines the token and it's role in the design system. It also changes the Docs output to always stringify the token value and fix the filePath to be relative to the tokens directory.

### CODE REVIEW

**Review by jcfranco — APPROVED** [review]
LGTM! 🕹️👾

I have a couple of questions regarding the attributes/schema field:

* Are these automatically populated from Figma? If not, what is the process to determine the schema?
* To reduce duplication, would it possible to provide a common schema at the token parent level and have each token extend or override the parent schema?

**Comment by alisonailea** [comment]
> * Are these automatically populated from Figma? If not, what is the process to determine the schema?

Not at this time. Automated Figma variables integration is still TBD. However, the schema organization was developed with the designers and the organization of Figma variables should mirror tokens. However I'm waiting for Skye to get back so we can do a final review of the schema before merging.

I'm also very open to feedback from devs as well. This naming schema will likely have long term effects on the organization of things. If needed I can move this into draft until the next engineering discussions.

> * To reduce duplication, would it possible to provide a common schema at the token parent level and have each token extend or override the parent schema?

I like this. It lends itself to part of a larger typescript refactor I was considering.

**Pull request merged**
